registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -316,6 +317,7 @@
     "sha256:95dc1c66fb7eed65003ba8e3d39c4a79f7be48b3fdaf87b94cf5099264b4f588": ["v1.8.1"]
     "sha256:0675084e31ec477ecc97c0c4f590fdea2fd7ffe399076dcde57996f6a8034555": ["v1.8.10"]
     "sha256:22bdc9893d81a61499607b3a97c48745ab85befafee688aa249e283e8ae0cccd": ["v1.8.11"]
+    "sha256:42be1448d8504e09a8ed6d7642451677ceb73428c60bce033d978ca60b692110": ["v1.8.12"]
```

Comment:
`42be1448d850` matches the SHA for clusterctl v1.8.12 in the [staging repository](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/clusterctl@sha256:42be1448d8504e09a8ed6d7642451677ceb73428c60bce033d978ca60b692110/details). 👍🏻